dockers/docker-database/flush_unused_database

```diff
@@ -6,7 +6,7 @@ import time
 
 while(True):
     output = subprocess.Popen(['sonic-db-cli', 'PING'], stdout=subprocess.PIPE).communicate()[0]
```

Comment:
>Popen [](http://example.com/codeflow?start=24&length=5)

Thanks for the fix! I see the `text=True` could also fix this, and it is more readable. Could you instead use that optional parameter?

Reply:
Done. Description updated